nginx-hello-nonroot/html-version/Makefile

```diff
@@ -0,0 +1,15 @@
+VERSION=0.1
+PREFIX=nginxdemos/nginx-hello
+TAG=$(VERSION)
+
+all: build push test
+
+build:
+	@echo "Building image:"
+	docker build -t $(PREFIX):$(TAG) .
+push:
+	@echo "Pushing image:"
+	docker push $(PREFIX):$(TAG)
+test:
+	@echo "Running container:"
+	docker run -d -p 8080 --name nginx-hello $(PREFIX):$(TAG)
```

Comment:
I suggest adding ` --rm` prefix so that container is removed after it exists. Also, better not to have `-d` option - it is ok if the container runs in the foreground. This test target is only needed so that we can quickly run the container locally and once we exit it, the container is removed.
Also, could you map port 8080 to 8080, so that we don't need to look up a randomly assigned port when we want to connect to the container?